### PR TITLE
add experiment settings

### PR DESCRIPTION
**Pull request by FionaC08**

### CODE REVIEW

File: configs/assignments/default.yaml [hunk 1]
```diff
@@ -5,11 +5,12 @@ concurrency:
     dbbench-std: 5
     os-std: 5
   agent:
-    gpt-3.5-turbo-0613: 5
+    # gpt-3.5-turbo-0613: 5
+    Llama-2-7b:5
```
Comment (EYH0602) [inline configs/assignments/default.yaml:9]: we are using vicuna-7b now

**Review by EYH0602 — CHANGES_REQUESTED** [review]

File: data/os_interaction/data/6/avalibillity.json [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (EYH0602) [inline data/os_interaction/data/6/avalibillity.json]: this file is empty?

**Review by EYH0602 — CHANGES_REQUESTED** [review]

**Comment by FionaC08** [comment]
Forgot to upload it. Already fixed it.

File: configs/agents/fs_agent.yaml [hunk 3]
```diff
@@ -2,10 +2,18 @@ default:
...
     max_new_tokens: 512
     temperature: 0
 
+llama-2-70b:
+  parameters:
+    model_name: "Llama-2-70b-v0"
+
+llama-2-7b:
+  parameters:
+    model_name: "Llama-2-7b-chat-hf"
```
Comment (EYH0602) [inline configs/agents/fs_agent.yaml]: do we still need config for Llama?

Comment (FionaC08) [reply configs/agents/fs_agent.yaml]: Nope. Switched to use gpt3.5 instead.

Comment (EYH0602) [reply configs/agents/fs_agent.yaml]: then you can remove this

**Comment by FionaC08** [comment]
ok！

**Review by EYH0602 — APPROVED** [review]

**Pull request merged**